app/models/physical_server.rb

```diff
@@ -6,6 +6,7 @@ class PhysicalServer < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::PhysicalInfraManager"
 
   default_value_for :enabled, true
+  has_one :host, :foreign_key => "service_tag", :primary_key => "serial_number"
```

Comment:
@blomquisg I agree with @rodneyhbrown7, because the in real life a "host" is installed into a "physical server", not the other way around. By the way, in real life (also) there is the possible of exists a "blank" physical server, with no hypervisor installed(which means that we'll not have a host context to this physical server).